test/CowAmmModule/ReplaceAmm.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {CowAmmModuleTestHarness, CowAmmModule, ConstantProduct} from "./CowAmmModuleTestHarness.sol";
+import {FallbackManager} from "lib/composable-cow/lib/safe/contracts/Safe.sol";
+
+abstract contract ReplaceAmmTest is CowAmmModuleTestHarness {
+    function testReplaceAmmWhenOneExists() public {
+        setUpDefaultCowAmm();
+
+        bytes32 previousOrderHash = cowAmmModule.activeOrders(safe);
```

Comment:
I'd check here to make sure the test expectations are met and we're indeed testing the replacement of an existing order.

```suggestion
        bytes32 previousOrderHash = cowAmmModule.activeOrders(safe);
        require(previousOrderHash != bytes32(0), "no order created in setup");
```